core/src/tangram.h

```diff
@@ -11,12 +11,27 @@ namespace Tangram {
 
 class DataSource;
 
+enum LabelType {
+    icon,
+    text,
+};
+
 struct TouchItem {
     std::shared_ptr<Properties> properties;
     float position[2];
     float distance;
```

Comment:
Doesn't seem like `distance` should be here any more - it's always zero for a feature result so we could make it a member of the label result instead. Or maybe we don't care that much about it?